server_addon/maya/server/settings/publishers.py

```diff
@@ -309,6 +482,73 @@ class ExtractAlembicModel(BaseSettingsModel):
         description="List of attribute prefixes for attributes that will be "
                     "included in the alembic export.",
     )
+    attr: str = SettingsField(
+        title="Custom Attributes",
+        placeholder="attr1;attr2",
+        description=(
+            "Attributes matching by name will be included in the Alembic "
+            "export. Attributes should be separated by semi-colon `;`"
+        )
+    )
+    attrPrefix: str = SettingsField(
+        title="Custom Attributes Prefix",
+        placeholder="prefix1;prefix2",
+        description=(
+            "Attributes starting with these prefixes will be included in the "
+            "Alembic export. Attributes should be separated by semi-colon `;`"
+        )
+    )
+    userAttr: str = SettingsField(
+        title="User Attr",
+        placeholder="attr1;attr2",
+        description=(
+            "Attributes matching by name will be included in the Alembic "
+            "export. Attributes should be separated by semi-colon `;`"
+        )
+    )
+    userAttrPrefix: str = SettingsField(
+        title="User Attr Prefix",
+        placeholder="prefix1;prefix2",
+        description=(
+            "Attributes starting with these prefixes will be included in the "
+            "Alembic export. Attributes should be separated by semi-colon `;`"
+        )
+    )
+    melPerFrameCallback: str = SettingsField(
+        title="Mel Per Frame Callback",
+        description=(
+            "When each frame (and the static frame) is evaluated the string "
+            "specified is evaluated as a Mel command."
+        )
+    )
+    melPostFrameCallback: str = SettingsField(
+        title="Mel Post Frame Callback",
+        description=(
+            "When the translation has finished the string specified is "
+            "evaluated as a Mel command."
+        )
+    )
```

Comment:
I suspect this needs to be post job callback?
```suggestion
    melPostJobCallback: str = SettingsField(
        title="Mel Post Job Callback",
        description=(
            "When the translation has finished the string specified is "
            "evaluated as a Mel command."
        )
    )
```

also see: https://github.com/alembic/alembic/blob/master/maya/AbcExport/AbcExport.cpp#L449C48-L449C66